src/gameState.js

```diff
@@ -52,6 +62,9 @@ var getDefault = (function () {
                     });
                 });
             };
+            Object.assign(defaultGameStateProxy, {
```

Comment:
Optional; is there a reason to not define it inline like the other methods here?

Reply:
Not really, I can fix it.